extensions/standard-processors/processors/TailFile.h

```diff
@@ -91,9 +99,12 @@ class TailFile : public core::Processor {
   static core::Property RecursiveLookup;
   static core::Property LookupFrequency;
   static core::Property RollingFilenamePattern;
+  static core::Property InitialStartPosition;
   // Supported Relationships
   static core::Relationship Success;
 
+  static const std::set<std::string> INITIAL_START_POSITIONS;
```

Comment:
is this used somewhere?

Reply:
No, I left it there by accident when fixing the comment, thanks! Removed in 393f540a9b21e7a3a8d8de78e3040519bac244af